scipy/stats/_continuous_distns.py

```diff
@@ -5851,6 +6081,11 @@ class ncx2_gen(rv_continuous):
     def _argcheck(self, df, nc):
         return (df > 0) & (nc >= 0)
```

Comment:
Noticed that it says nc > 0 in the documentation, but we can fix that another time if you prefer.